### DISCUSSION TITLE
Which ORM would you recommend using with this?

### DISCUSSION THREAD

**Discussion by ckcollab** [cirospaciari/socketify.py#91] [Q&A]
First of all thanks for all of the impressive work on this, pretty inspiring!

What's your perspective on ORMs and which would you use on socketify.py projects, if any?

**Comment by cirospaciari** [comment]
Hey there, I want to implement connectors to some DBs like postgres, mariadb/mysql and redis, you can keep track on https://github.com/cirospaciari/socketify.py/issues/65 and https://github.com/cirospaciari/socketify.py/issues/71

ORM will come in the future for sure, I recommend you use the best for your use case.
If you want to use it with PyPI check the best ORM + connector for it and the same for CPython.
I did not benchmark any current implementation yet (but I will in the future) so I don't have any recommendations for now, sorry.

The goal of socketify is one day to provide all this for you.
